docs/linux/external_fuzzing_usbip.md

```diff
@@ -0,0 +1,35 @@
+# **USB/IP Fuzzing for Linux Kernel**
+
+Syzkaller supports fuzzing the Linux kernel USB/IP subsystem externally. We can set up a virtual network and send USB/IP packets to the client kernel as they are being received from an external server.
+USB/IP fuzzing needs USB/IP configurations to be enabled. You can find the list in the configurations part.
+
+For the moment, USB/IP Fuzzing consists of fuzzing the client's side and can be explained in 2 main parts:
+1. USB/IP pseudo-syscalls.
+2. Syzkaller descriptions.
+
+### **Configurations**
+
+Following configurations should be enabled for USB/IP.
+
+```
+CONFIG_USBIP_CORE=y
+CONFIG_USBIP_VHCI_HCD=y
+CONFIG_USBIP_VHCI_HC_PORTS=8
+CONFIG_USBIP_VHCI_NR_HCS=16
```

Comment:
`8` should be enough? We typically have `8` procs.

Reply:
Yes, I just took upstream-kasan.config's for convenience.

Reply:
Ah, OK. Either one will work, but matching the value we have in executor with the value here looks more reasonable.